### DISCUSSION TITLE
Awful LORA results (blurry)

### DISCUSSION THREAD

**Discussion by Pilot321-ui** [lllyasviel/stable-diffusion-webui-forge#1424] [Q&A] [answered]
Everything in default, Diffusion in Low Bits in Automatic, model flux1-dev-Q4_0.gguf.

Absolutely blurry results.

It's definitely the LORA, because without it, the image looks just fine. Something is off here, I wasn't getting such awful results before.

Tried a couple of Flux LORAs from Civitai, same blurry result.

Why?
![00019-3391284850](https://github.com/user-attachments/assets/2af62af6-f14c-45e2-99c4-bd551d35370d)

**EDIT** I just reloaded Forge and everything was running better later. I don't know exactly what was happening.

**Comment by ostap667inbox** [comment]
Set lower weight for LoRA with GGUF model.
Your example is very typical of what LoRA overweight looks like.

**Reply by Pilot321-ui** [reply, marked as answer]
Yeah I don't know what happened there, I just reloaded Forge and everything is running way better now.

**Comment by freakstritch** [comment]
I solved the blurry lora issue by selecting Automatic FP16 LORA - for the diffusion in low bits - not just Automatic.

**Reply by evanheckert** [reply]
This solved it for me, too. Thank you!

**Reply by sjorspa** [reply]
I had a same issue were a Lora would not work at all, with automatic it did not work, after setting it to bnb-nf4 (fp16 LoRA) it worked.
I first thought it was on the training, but after creating 3 loras I finally found it :D

**Comment by Weatherby43** [comment]
I was fine, updated over the last couple days (happened on restart) and now my lora images look like crap too. Way over cooked and or just like a 70s screen capture off tv...artifacting bad. Nothing changed prompt/weight wise. Out if the blue images just went from crip and clear to artifacts all over and poor quality.  I suspect something beyond too high lora weight, or it would have been doing this all along. And it's every lora causes the problem.

**Reply by freakstritch** [reply]
do you have FP16 LORA selected?

**Comment by ostap667inbox** [comment]
After the updates of the last few days forge started to produce a lot of blurry pictures, regardless of whether LoRA is additionally enabled or not
![00006-876985463](https://github.com/user-attachments/assets/14da9cd2-0242-44e7-b8e3-dd755cd801de)
![00007-4007785325](https://github.com/user-attachments/assets/c7c4e5fc-efd1-4856-b210-237c476772fa)

**Reply by wesux** [reply]
I'm noticing the same thing. I can get a few crisp pictures then they all go blurry. Also blurry when using Lora right from the start.

**Reply by Juqowel** [reply]
If you are using "white background" prompt - looks like the problem with FLUX itself.
[https://github.com/black-forest-labs/flux/issues/107](https://github.com/black-forest-labs/flux/issues/107)

**Comment by OFF-AI** [comment]
That does'nt look blurry to me. It looks like the wrong CFG scale. It could be that you need to consider if its a distilled or dedestilled Lora (or checkpoint).
